report/how_to.md

```diff
@@ -0,0 +1,56 @@
+# User Guide
+
+## How to use this book
+
+### Navigating tags
+
+Each case is categorized by a collection of relevant tags. A specific tag's cases (such as `2D`) can be navigated through the [🏷 Tags](./_tags/tagsindex.md) index.
+
+### Executing and editing code
+
+Press the {fas}`rocket` button on the toolbar, then the {fas}`play` button to edit and run the code.
+
+```{note}
+This might take a while to load after new releases of the book.
+```
+
+## Converting Markdown files to Jupyter notebooks
```

Comment:
Users can also download pages as jupyter notebooks by clicking the download button up top